### ISSUE TITLE
Move `cliff` and `cli` to devDependencies

### ISSUE THREAD

**Issue by rmadsen** [beaugunderson/ip-address#41]
First of all, this looks like a great library with a ton of useful utility functions! I was wondering if we could move `cliff` and `cli` to the `devDependencies` section of `package.json`. It looks like the only places those modules are used in are two scripts in `bin`, and while (presumably) `bin` scripts' `requires` should show up in dependencies, it doesn't look like either `ipv6.js` or `ipv6grep.js` look like they are meant for public consumption. Perhaps those `bin` scripts should be moved to `examples/` as well?

In any case, it would be nice to not have to install the following trees:

```
  ├─┬ cli@0.11.3
  │ └─┬ glob@7.1.1
  │   └── minimatch@3.0.3
  ├─┬ cliff@0.1.10
  │ ├── colors@1.0.3
  │ ├── eyes@0.1.8
  │ └─┬ winston@0.8.3
  │   ├── async@0.2.10
  │   ├── colors@0.6.2
  │   ├── cycle@1.0.3
  │   ├── pkginfo@0.3.1
  │   └── stack-trace@0.0.9
```

If you agree, I'm happy to submit a PR.

**Comment by beaugunderson** [comment]
yes, those scripts can be moved to a new package :)

---

**Comment by beaugunderson** [comment]
@rmadsen removed now! published in 5.8.3
